spec/features/users_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Users', type: :feature do
```

Comment:
Please name this spec `User Registration` or  `User registration process` or similar. All names should be clear